tools/c7n_gcp/tests/data/flights/instance-group-manager-delete/get-compute-v1-projects-mitrop-custodian-aggregated-instanceGroupManagers_1.json

```diff
@@ -0,0 +1,1038 @@
+              "key": "scope"
+            }
+          ]
+        }
+      }, 
+      "regions/us-east1": {
+        "warning": {
+          "message": "There are no results for scope 'regions/us-east1' on this page.", 
+          "code": "NO_RESULTS_ON_PAGE", 
+          "data": [
+            {
+              "value": "regions/us-east1", 
+              "key": "scope"
+            }
+          ]
+        }
+      }, 
+      "regions/europe-north1": {
+        "warning": {
+          "message": "There are no results for scope 'regions/europe-north1' on this page.", 
+          "code": "NO_RESULTS_ON_PAGE", 
+          "data": [
+            {
+              "value": "regions/europe-north1", 
+              "key": "scope"
+            }
+          ]
+        }
+      }, 
+      "regions/us-east4": {
+        "warning": {
+          "message": "There are no results for scope 'regions/us-east4' on this page.", 
+          "code": "NO_RESULTS_ON_PAGE", 
+          "data": [
+            {
+              "value": "regions/us-east4", 
+              "key": "scope"
+            }
+          ]
+        }
+      }, 
+      "zones/asia-south1-a": {
+        "warning": {
+          "message": "There are no results for scope 'zones/asia-south1-a' on this page.", 
+          "code": "NO_RESULTS_ON_PAGE", 
+          "data": [
+            {
+              "value": "zones/asia-south1-a", 
+              "key": "scope"
+            }
+          ]
+        }
+      }, 
+      "zones/northamerica-northeast1-a": {
+        "warning": {
+          "message": "There are no results for scope 'zones/northamerica-northeast1-a' on this page.", 
+          "code": "NO_RESULTS_ON_PAGE", 
+          "data": [
+            {
+              "value": "zones/northamerica-northeast1-a", 
+              "key": "scope"
+            }
+          ]
+        }
+      }, 
+      "zones/australia-southeast1-b": {
+        "warning": {
+          "message": "There are no results for scope 'zones/australia-southeast1-b' on this page.", 
+          "code": "NO_RESULTS_ON_PAGE", 
+          "data": [
+            {
+              "value": "zones/australia-southeast1-b", 
+              "key": "scope"
+            }
+          ]
+        }
+      }, 
+      "regions/asia-south1": {
+        "warning": {
+          "message": "There are no results for scope 'regions/asia-south1' on this page.", 
+          "code": "NO_RESULTS_ON_PAGE", 
+          "data": [
+            {
+              "value": "regions/asia-south1", 
+              "key": "scope"
+            }
+          ]
+        }
+      }, 
+      "zones/us-west2-b": {
+        "warning": {
+          "message": "There are no results for scope 'zones/us-west2-b' on this page.", 
+          "code": "NO_RESULTS_ON_PAGE", 
+          "data": [
+            {
+              "value": "zones/us-west2-b", 
+              "key": "scope"
+            }
+          ]
+        }
+      }, 
+      "zones/northamerica-northeast1-b": {
```

Comment:
let's strip aggregated pages of empty results content, it bloats the pr, and slows tests.

Reply:
Done.